azure/template.json

```
@@ -681,6 +685,10 @@
                                 "name": "SKYLIGHT_ENABLE",
                                 "value": "[parameters('skylightEnable')]"
                             },
+                            {
+                                "name": "SKYLIGHT_ENV",
+                                "value": "[parameters('skylightEnv')]"
```

```suggestion
                                "value": "[parameters('environment')]"
```
Can we just reuse the `environment` parameter which already holds the value for the appropriate environment `qa`, `staging` etc ?
https://github.com/DFE-Digital/apply-for-teacher-training/blob/1d665b3e4c06b1bb74f2b05924add5ba3419dabd/azure/template.json#L37